packages/paste-website/src/components/site-wrapper/sidebar/SidebarNavigation.tsx

```diff
@@ -134,6 +134,9 @@ const SidebarNavigation: React.FC = () => {
             <SidebarAnchor level={2} to={`${SidebarCategoryRoutes.CONTRIBUTING}/patterns`}>
               Patterns
             </SidebarAnchor>
+            <SidebarAnchor level={2} to={`${SidebarCategoryRoutes.CONTRIBUTING}/components`}>
```

Comment:
Do we have a particular order we want the Contributing pages to be in?

Reply:
I don't necessarily have an opinion on this but @gloriliale suggested alphabetical, what do you think?

Reply:
We used alphabetical in the IA changes doc so that works!